bundle/edu.gemini.horizons.api/src/main/scala/edu/gemini/horizons/server/backend/HS2.scala

```diff
@@ -160,8 +162,15 @@ object HorizonsService2 {
         TIME_DIGITS      -> FRACTIONAL_SEC
       )
 
+    val replyType = target match {
+      case Comet(des)            => HorizonsReply.ReplyType.COMET
+      case AsteroidNewStyle(des) => HorizonsReply.ReplyType.MINOR_OBJECT
+      case AsteroidOldStyle(num) => HorizonsReply.ReplyType.MINOR_OBJECT
+      case MajorBody(num)        => HorizonsReply.ReplyType.MAJOR_PLANET
+    }
```

Comment:
I'm not sure whether this is the correct mapping, or if there is a correct mapping.  It appears to be irrelevant since we just ignore everything but the ephemeris data in the `HorizonsReply` we get back from `CgiReplyBuilder`.